lq1/accessibility.txt

```diff
@@ -0,0 +1,11 @@
+=========================================
+Useful Console Commands for Accessibility
+=========================================
+
+gl_zfix 1 (to prevent texture flickering aka "z fighting"; only works in Ironwail, but not FTEQCC)
```

Comment:
Do you mean to say FTEQW, not FTEQCC?

Reply:
Dammit! I keep doing that! I removed that line anyway since I was able to make the command FTEQW compatible.